src/tests/ftest/util/environment_utils.py

```diff
@@ -178,6 +181,12 @@ def set_defaults(self, logger, servers=None, clients=None, provider=None, insecu
             self.agent_user = self._default_agent_user()
         if self.systemd_library_path is None:
             self.systemd_library_path = self._default_systemd_library_path()
+        if self.control_config is None:
+            self.control_config = self._default_control_config()
```

Comment:
As these keep growing, I think it would be cleaner to do these inline
```
self.control_config = os.path.join(self.log_dir, "configs", "daos_control.yml")
```
Instead of defining a helper method that is used in one location that just returns a string

Reply:
It would clean things up a bit, but as it's just cleanup it could be handled in another PR.